docs/docs/references/configuration/config-file.md

```diff
@@ -179,346 +79,558 @@ cache:
-    - misconfig
-    - secret      
-  
-  docker:
-    # Same as '--docker-host'
-    # Default is empty
-    host: 
-  
   podman:
     # Same as '--podman-host'
     # Default is empty
-    host: 
-```
+    host:
 
-## Vulnerability Options
-Available with vulnerability scanning
+  # Same as '--removed-pkgs'
+  # Default is false
+  removed-pkgs: false
+
+  # Same as '--image-src'
+  # Default is docker, containerd, podman, remote
+  source:
+   - docker
+   - containerd
+   - podman
+   - remote
+
+```
+## Kubernetes options
 
 ```yaml
-vulnerability:
-  # Same as '--ignore-unfixed'
+kubernetes:
+  # Same as '--burst'
+  # Default is 10
+  burst: 10
+
+  # Same as '--disable-node-collector'
   # Default is false
-  ignore-unfixed: false
+  disableNodeCollector: false
 
-  # Same as '--ignore-unfixed'
+  exclude:
+    # Same as '--exclude-nodes'
+    # Default is []
+    nodes: empty
+
+    # Same as '--exclude-owned'
+    # Default is false
+    owned: false
+
+  # Same as '--exclude-kinds'
+  # Default is []
+  excludeKinds: empty
+
+  # Same as '--exclude-namespaces'
+  # Default is []
+  excludeNamespaces: empty
+
+  # Same as '--include-kinds'
+  # Default is []
+  includeKinds: empty
+
+  # Same as '--include-namespaces'
+  # Default is []
+  includeNamespaces: empty
+
+  # Same as '--k8s-version'
   # Default is empty
-  ignore-status: 
-    - end_of_life
+  k8s-version:
 
-  # Same as '--vex'
+  # Same as '--kubeconfig'
   # Default is empty
-  vex:
-    - path/to/vex/file
-    - repo
+  kubeconfig:
 
-  # Same as '--skip-vex-repo-update'
-  # Default is false
-  skip-vex-repo-update: true
-```
+  node-collector:
+    # Same as '--node-collector-imageref'
+    # Default is ghcr.io/aquasecurity/node-collector:0.3.1
+    imageref: ghcr.io/aquasecurity/node-collector:0.3.1
 
-## License Options
-Available with license scanning
+    # Same as '--node-collector-namespace'
+    # Default is trivy-temp
+    namespace: trivy-temp
 
-```yaml
-license:
-  # Same as '--license-full'
+  # Same as '--qps'
+  # Default is 5
+  qps: 5
+
+  # Same as '--skip-images'
   # Default is false
-  full: false
+  skipImages: false
 
-  # Same as '--ignored-licenses'
-  # Default is empty
-  ignored:
-    - MPL-2.0
-    - MIT
+  # Same as '--tolerations'
+  # Default is []
+  tolerations: empty
 
+```
+## License options
+
+```yaml
+license:
   # Same as '--license-confidence-level'
   # Default is 0.9
   confidenceLevel: 0.9
 
-  # Set list of forbidden licenses
-  # Default is https://github.com/aquasecurity/trivy/blob/164b025413c5fb9c6759491e9a306b46b869be93/pkg/licensing/category.go#L171
+  # Default is AGPL-1.0, AGPL-3.0, CC-BY-NC-1.0, CC-BY-NC-2.0, CC-BY-NC-2.5, CC-BY-NC-3.0, CC-BY-NC-4.0, CC-BY-NC-ND-1.0, CC-BY-NC-ND-2.0, CC-BY-NC-ND-2.5, CC-BY-NC-ND-3.0, CC-BY-NC-ND-4.0, CC-BY-NC-SA-1.0, CC-BY-NC-SA-2.0, CC-BY-NC-SA-2.5, CC-BY-NC-SA-3.0, CC-BY-NC-SA-4.0, Commons-Clause, Facebook-2-Clause, Facebook-3-Clause, Facebook-Examples, WTFPL
```

Comment:
I think it's not convenient to work with such a large number of values ​​(GH pages use horizontal scrolling)
I don't like hardcoding, but maybe we should override default values with GH links (as is done now)?

Reply:
as we told offline, my thought about usecases is next:
it's convenient to copy/paste default values from docs and then to customize it.
maybe I'm wrong. 

I thought the hardcoded link was added to show actual state for default values.
Now the actual state is added automatically.

Reply:
If we use all default values ​​in `yaml` struct - it will be easier for users to get default values ​​from `yaml` struct than to copy them from comment.
I suggested limiting the number of default values ​​[here](https://github.com/aquasecurity/trivy/pull/7261#discussion_r1701274069).

But maybe a better way is to remove `Default is ...` comment, but use all default values ​​in the `yaml` struct.

Reply:
ok, let's try.
removed default values.

Reply:
Let's add information before `Global Options` - that `yaml` files below contain default values.

Reply:
done

Reply:
Great. Thanks!